### PR TITLE
LIME-202: Add user info card

### LINKED ISSUE #202: Add User info
Image

### PR DESCRIPTION
**Pull request by SergiiTernytskyi**
Added user info card

### CODE REVIEW

**Comment by SergiiTernytskyi** [comment]
user-info

File: frontend/src/bundles/common/components/user-info-card/user-info-card.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+import { ComponentSize } from '../../enums/component-size.enum.js';
+import { AchievementCard, Button, ButtonVariant } from '../components.js';
+
+type Properties = {
+    name: string;
+    image: string;
+    isActive: boolean;
+    onClick: () => void;
+    achievements?: [];
```
Comment (anton-otroshchenko) [inline frontend/src/bundles/common/components/user-info-card/user-info-card.tsx]: Don't we have type for achievement?

File: frontend/src/bundles/common/helpers/date-converter/date-converter.ts
Comment (anton-otroshchenko) [inline frontend/src/bundles/common/helpers/date-converter/date-converter.ts:1]: add .helper to the file name

**Review by anton-otroshchenko — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/helpers/date-converter/date-converter.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { DateObject } from 'react-multi-date-picker';
+
+const dateConverter = (date: string): string => {
+    const convertedDate = new DateObject(date);
+
+    return convertedDate.format('dddd, MMMM DD | HH:mm A');
+};
+
```
Comment (dimapopovych) [inline frontend/src/bundles/common/helpers/date-converter/date-converter.ts]: rename to formatDate or something similar, function name should start with verb. Also we already have similar helper, maybe it's worth to create formatDate(date, format) where format will be value from enum or so.

**Review by dimapopovych — CHANGES_REQUESTED** [review]

**Comment by SergiiTernytskyi** [comment]
Screenshot_13

File: frontend/src/bundles/common/components/user-info-card/user-info-card.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+const UserInfoCard: React.FC<Properties> = ({
+    name,
+    image,
+    isActive = true,
+    achievements = [],
+}) => {
+    const navigate = useNavigate();
+
+    const onViewAllClick = useCallback((): void => {
```
Comment (dimapopovych) [inline frontend/src/bundles/common/components/user-info-card/user-info-card.tsx]: handlers should have handle prefix

**Review by dimapopovych — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/user-info-card/user-info-card.tsx (hunk 3 above)
Comment (SergiiTernytskyi) [reply frontend/src/bundles/common/components/user-info-card/user-info-card.tsx]: Thats wright

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**